### PR TITLE
Add a loading state to Cart i2

### LINKED ISSUE #4964: Cart i2: block has no skeleton and the Empty cart is showed until the block loads
## Describe the bug
There is no loading state for the block that shows the cart preview. Because of this, the empty cart view is shown until the block loads.

## Expected behaviour
Show a preview until the block loads.

This was discovered while working on #4793

### PR DESCRIPTION
**Pull request by ralucaStan**
We use CSS and the is-loading class to hide the empty state and show a loading state for the filled state.

Fixes #4964 

How to test the changes in this Pull Request:

1. Add Cart i2 to a page
2. In the front-end, notice the place holder that appears until the block loads
3. Switch to a mobile viewport 
4. Notice how the placeholder has a mobile layout

### CODE REVIEW

**Review by mikejolley — APPROVED** [review]
Tested and looks good 👍🏻 

Since this is smaller than scope from the i1-i2 migration, I'd suggest holding off merging and rebasing once thats merged to trunk.

**Comment by ralucaStan** [comment]
@mike agree on the merge part

**Review by senadir — APPROVED** [review]

**Pull request merged**
